sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_convert_to_and_from_dict_async.py

```diff
@@ -0,0 +1,128 @@
+# coding: utf-8
```

Comment:
thought - this feature was available in the GA, should we have a sample in v3.1 directory as well?

Reply:
Working on this....hit a snag with JSON serializing dates :/

Reply:
maybe an opportunity to use `AzureJSONEncoder` from azure.core?

Reply:
Oh yes, I shall try that 🙌